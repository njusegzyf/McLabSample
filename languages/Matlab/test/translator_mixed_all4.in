function q
function [u v] = f(y)
x = [1 : 5 : y]
x = x .* [1 2 3 4]
cmd end a'a'a'a'a