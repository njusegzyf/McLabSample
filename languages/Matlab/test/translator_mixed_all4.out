>>>> Destination Language -> Source Language
[1, 1] -> (1, 1)
[1, 2] -> (1, 2)
[1, 3] -> (1, 3)
[1, 4] -> (1, 4)
[1, 5] -> (1, 5)
[1, 6] -> (1, 6)
[1, 7] -> (1, 7)
[1, 8] -> (1, 8)
[1, 9] -> (1, 9)
[1, 10] -> (1, 10)
[1, 11] -> (1, 11)
[2, 1] -> (1, 11)
[2, 2] -> (1, 11)
[2, 3] -> (1, 11)
[2, 4] -> (1, 11)
[3, 1] -> (1, 11)
[4, 1] -> (2, 1)
[4, 2] -> (2, 2)
[4, 3] -> (2, 3)
[4, 4] -> (2, 4)
[4, 5] -> (2, 5)
[4, 6] -> (2, 6)
[4, 7] -> (2, 7)
[4, 8] -> (2, 8)
[4, 9] -> (2, 9)
[4, 10] -> (2, 10)
[4, 11] -> (2, 11)
[4, 12] -> (2, 11)
[4, 13] -> (2, 12)
[4, 14] -> (2, 13)
[4, 15] -> (2, 14)
[4, 16] -> (2, 15)
[4, 17] -> (2, 16)
[4, 18] -> (2, 17)
[4, 19] -> (2, 18)
[4, 20] -> (2, 19)
[4, 21] -> (2, 20)
[4, 22] -> (2, 21)
[4, 23] -> (2, 22)
[5, 1] -> (1, 11)
[6, 1] -> (1, 11)
[7, 1] -> (3, 1)
[7, 2] -> (3, 2)
[7, 3] -> (3, 3)
[7, 4] -> (3, 4)
[7, 5] -> (3, 5)
[7, 6] -> (3, 6)
[7, 7] -> (3, 7)
[7, 8] -> (3, 8)
[7, 9] -> (3, 9)
[7, 10] -> (3, 10)
[7, 11] -> (3, 11)
[7, 12] -> (3, 12)
[7, 13] -> (3, 13)
[7, 14] -> (3, 14)
[7, 15] -> (3, 15)
[7, 16] -> (3, 16)
[8, 1] -> (4, 1)
[8, 2] -> (4, 2)
[8, 3] -> (4, 3)
[8, 4] -> (4, 4)
[8, 5] -> (4, 5)
[8, 6] -> (4, 6)
[8, 7] -> (4, 7)
[8, 8] -> (4, 8)
[8, 9] -> (4, 9)
[8, 10] -> (4, 10)
[8, 11] -> (4, 11)
[8, 12] -> (4, 11)
[8, 13] -> (4, 12)
[8, 14] -> (4, 13)
[8, 15] -> (4, 13)
[8, 16] -> (4, 14)
[8, 17] -> (4, 15)
[8, 18] -> (4, 15)
[8, 19] -> (4, 16)
[8, 20] -> (4, 17)
[8, 21] -> (4, 18)
[8, 22] -> (4, 19)
[9, 1] -> (5, 1)
[9, 2] -> (5, 2)
[9, 3] -> (5, 3)
[9, 4] -> (5, 5)
[9, 5] -> (5, 5)
[9, 6] -> (5, 5)
[9, 7] -> (5, 6)
[9, 8] -> (5, 7)
[9, 9] -> (5, 7)
[9, 10] -> (5, 7)
[9, 11] -> (5, 9)
[9, 12] -> (5, 9)
[9, 13] -> (5, 11)
[9, 14] -> (5, 13)
[9, 15] -> (5, 15)
[9, 16] -> (5, 17)
[9, 17] -> (5, 17)
[9, 18] -> (5, 17)
[9, 19] -> (5, 17)
[10, 1] -> (5, 17)
[10, 2] -> (5, 17)
[10, 3] -> (5, 17)
>>>> Translated Text
function q
end

function [u, v] = f(y)


x = [1 : 5 : y]
x = x .* [1, 2, 3, 4]
cmd('end','aaaaa')
end